pyaedt/modules/Boundary.py

```diff
@@ -1592,6 +1606,32 @@ def join_parallel(self, sources, matrix_name=None, join_name=None):
         )
 
 
+class MaxwellMatrix(object):
+    def __init__(self, app, parent_name):
+        self._app = app
+        self._parent_matrix = parent_name
+        self.names = []
+        self.sources = []
+        if self._app.solution_type == "EddyCurrent":
+            self.names = self._app.odesign.GetChildObject("Parameters").GetChildObject(parent_name).GetChildNames()
+            for m in self.names:
+                sources = (
+                    self._app.odesign.GetChildObject("Parameters")
+                    .GetChildObject(parent_name)
+                    .GetChildObject(m)
+                    .GetChildNames()
+                )
+                for s in sources:
+                    excitations = (
+                        self._app.odesign.GetChildObject("Parameters")
+                        .GetChildObject(parent_name)
+                        .GetChildObject(m)
+                        .GetChildObject(s)
+                        .GetPropValue("Source")
+                    )
+                    self.sources.append({m: {s: excitations}})
+
+
 class FieldSetup(BoundaryCommon, object):
     """Manages Far Field and Near Field Component data and execution.
```

Comment:
```suggestion
    """Manages far field and near field component data and execution.
```
Capitalize only proper nouns.